scripts/env

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+_set_prefix() {
+    export PREFIX=""
+
+    if [ -d "venv" ]; then
+        export PREFIX="venv/bin/"
+
+        if [[ "$OSTYPE" == "cygwin" || $OSTYPE == "msys" ]]; then
+            # Windows.
+            export PREFIX="venv/Scripts/"
+        fi
+    fi
+}
+
+set_env() {
+    _set_prefix
+}
+
+set_env
```

Comment:
What's the reason for doing this? i.e. defining another function that simply calls another function?

Reply:
No particular functional reason, this is mostly for readability. I could have named the script `scripts/prefix` but figured it would be better to have a general script for setting up environment variables, hence `scripts/env`. Having the entrypoint be `set_prefix` didn't feel consistent, hence this shim function. :)

Reply:
Alright 👍